module.yml

```diff
@@ -4,5 +4,5 @@ description:
 license: ASL 2.0
 copyright: 2021 Inmanta
 name: terraform
-version: 1.0.4
+version: 1.0.5.dev0
```

Comment:
Did you get this from `inmanta module commit`? If so, could you tell me on which core version?

Reply:
I didn't, I just cheated a little bit and manually set this.  I expect this won't cause issue as I will do a release for this branch anyway.

Reply:
Indeed